lib/msal-angular/docs/v2-docs/1.x-2.x-upgrade-guide.md

```diff
@@ -43,10 +43,14 @@ See the [updated sample](https://github.com/AzureAD/microsoft-authentication-lib
 
 ### API changes
 * The `acquireToken` and `login` methods now take different request objects as parameters. See the [msal.service.ts](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/src/msal.service.ts) for details.
-* `getAccount()` is now `getAccountByUsername()`. There is also a `getAllAccounts()` method available.
 * Broadcast events now emit an `EventMessage` object, instead of just strings. See the [Angular sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.component.ts) for an example of how to implement.
 * Applications using `Redirect` methods must implement the `handleRedirectObservable` method (and have it run on every page load), which will capture the result of redirect operations. See the [Angular sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/home/home.component.ts) for an example of how to implement.
 
+### Accounts
+* `getAccount()` is now `getAccountByUsername()` and is available on the MSAL instance. 
```

Comment:
We should be advertising `getAccountByLocalId` or `getAccountByHomeId` as the gold standard with `getAccountByUsername` being considered less reliable and for convenience only.

Reply:
Thanks! I changed the upgrade guide to recommend that.